### ISSUE TITLE
The script shouldn't let users choose between Release and Master

### ISSUE THREAD

**Issue by TheFrenchGhosty** [tmiland/Invidious-Updater#31]
The script currently ask the users what they want between Release and Master when Installing / Updating.

Releases are deprecated (I recently added a message [here](https://github.com/iv-org/invidious/releases) about it), and I don't think we'll ever do any more releases, so letting users choose to use a release is a bad idea.

This should only deploy invidious from master, and completely ignore the releases.

In short: this shouldn't be asked:

![script](https://user-images.githubusercontent.com/47571719/96903900-ef202800-1485-11eb-921c-2c53004df9bc.png)

**Comment by tmiland** [comment]
Removed the options. 👍

**Comment by TheFrenchGhosty** [comment]
Tested and working. Thanks.
